kidkode/src/main/webapp/script.js

```diff
@@ -178,27 +177,32 @@ $( window ).on( "load", function() {
 function showSlide(newStep, move) {
   // remove the end of quiz slide or current slide as active slide
   document.getElementById("end-slide").classList.remove("active-slide");
-  var toRemove = document.getElementById(currentStep);
-  toRemove.classList.remove("active-slide");
+  // if current step is null, the user has reloaded while on the final quiz page
+  if (currentStep != "null") {
+       let toRemove = document.getElementById(currentStep);
+       toRemove.classList.remove("active-slide");
+  }
   // if not at the end of quiz
-  if (newStep != null) {
+  if ((newStep  != "null" && newStep  != null)) {
```

Comment:
I have to change this tomorrow, somewhere in the code "null" and null are getting mixed up which is why I have this statement in multiple places (checking for "null" and null).